src/importexport/finale/tests/finale_tests.cpp

```
@@ -0,0 +1,252 @@
+
+#define MUSX_IMPORT_TEST_DISABLED(name) \
+    TEST_F(Finale_Tests, DISABLED_##name) { \
+        finaleImportTestRef(#name); \
+    }
+
+MUSX_IMPORT_TEST(arpeggios)
+MUSX_IMPORT_TEST(arpeggios2)
+MUSX_IMPORT_TEST(articulations1)
+MUSX_IMPORT_TEST_DISABLED(beamOverBarlines)
+MUSX_IMPORT_TEST(beamOverMultBarsGraceNotes)
+MUSX_IMPORT_TEST(beamOverNachschläge)
```

Cppcheck (2.21.0)

[error] The code contains unhandled character(s) (character code=195). Neither unicode nor extended ascii is supported.

(unhandledChar)